src/metabase/server/middleware/session.clj

```diff
@@ -68,6 +69,38 @@
                                                        metabase-embedded-session-cookie
                                                        metabase-session-timeout-cookie]))
 
+(def ^:private possible-session-cookie-samesite-values
+  #{:lax :none :strict nil})
+
+(defn- valid-session-cookie-samesite!
+  [value & [added-ex-data]]
+  (let [normalized-value (some-> value name str/lower-case keyword)]
```

Comment:
You're getting a clj-kondo error here due to the use of `str/lower-case`—we discourage it in favor of `metabase.util/lower-case-en` which will have consistent results regardless of your site locale. (`str/lower-case` will have inconsistent results if your locale is Turkish which has bitten us before.)

Seems like it this linter rule was ignored in the previous code but I don't see a good reason for that over just using our custom function.

Reply:
Nice, thanks for the heads up - sorry for missing the linter error!